doc/api/process.md

```diff
@@ -928,6 +928,63 @@ console.log(process.env.test);
 
 `process.env` is read-only in [`Worker`][] threads.
 
+## process.allowedEnvironmentNodeFlags
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Array}
+
+The `process.allowedEnvironmentNodeFlags` property is an array of flags
+allowable within the [`NODE_OPTIONS`][] environment variable.
+
+If Node.js was compiled *without* `NODE_OPTIONS` support (shown in
+[`process.config`][]), this array will contain what *would have been
+allowable*.
```

Comment:
This seems like undesirable behavior - what’s the reasoning for this?

Reply:
You can get the info this way, and combined with process config, get the empty array also. The reverse is not true.

Reply:
Mm fair enough.